src/Pages/MembershipApplication/MembershipForm.js

```diff
@@ -38,50 +47,66 @@ export default function MembershipForm(props) {
       addon: !usernameAvailable && (
         <p className='unavailable'>User already exists!</p>
       ),
-      handleChange: e => setEmail(e.target.value)
+      ifLeftEmpty: !email && submitted && (
```

Comment:
<img width="664" alt="Screen Shot 2020-11-02 at 8 41 03 PM" src="https://user-images.githubusercontent.com/50391828/97950568-24f9c200-1d4c-11eb-8ea5-3ca390371d98.png">
it doesnt let me submit with invalid emails, but the error message isn't specific. Could you change the error handling?

Reply:
I check if the email includes character '@', otherwise print out error message. In addition, ifLeftEmpty property is changed to ifRequirementsNotMet since that's more appropriate. Checking email input now have a separate function